### PR TITLE
[AF-1555] Upgrade to Wildfly14

### PR DESCRIPTION
**Pull request by adrielparedes**
## Description

There is a concern related to SNAPSHOT version that will be solved in jboss-integration-platform-bom

@tiagobento could you review this please?

## Related to:
* https://github.com/jboss-integration/jboss-integration-platform-bom/pull/427
* https://github.com/errai/errai/pull/363
* https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/843
* https://github.com/kiegroup/appformer/pull/483
* https://github.com/kiegroup/kie-wb-common/pull/2147
* https://github.com/kiegroup/jbpm/pull/1336
* https://github.com/kiegroup/jbpm-wb/pull/1206
* https://github.com/kiegroup/drools/pull/2086
* https://github.com/kiegroup/droolsjbpm-integration/pull/1583
* https://github.com/kiegroup/kie-wb-distributions/pull/821 
* https://github.com/kiegroup/kie-docs/pull/1108
* https://github.com/kiegroup/optaplanner/pull/437
* https://github.com/kiegroup/optaplanner-wb/pull/304

### CODE REVIEW

**Comment by tiagobento** [comment]
@adrielparedes 	So this PR will be updated to get rid of the SNAPSHOT version?

**Comment by kiereleaseuser2** [comment]
Build finished. 2761 tests run, 5 skipped, 0 failed.

File: errai-bom/pom.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
   <parent>
     <groupId>org.jboss.integration-platform</groupId>
     <artifactId>jboss-integration-platform-parent</artifactId>
-    <version>8.1.0.Final</version>
+    <version>8.2.0-SNAPSHOT</version>
```
[the remark this answers is not in the document]
Comment (jhrcek) [reply errai-bom/pom.xml]: @adrielparedes could you please address this comment - 2 usages of 8.2.0-SNAPSHOT version of IP BOM in this PR.

**Comment by kiereleaseuser2** [comment]
Build finished. No test results found.

**Comment by kiereleaseuser2** [comment]
Build finished. 2759 tests run, 5 skipped, 2 failed.

**Comment by adrielparedes** [comment]
jenkins retest this

**Comment by kiereleaseuser2** [comment]
Build finished. 2759 tests run, 5 skipped, 2 failed.

**Comment by kiereleaseuser2** [comment]
Build finished. 2759 tests run, 5 skipped, 2 failed.

**Comment by tiagobento** [comment]
@adrielparedes Hi! Did you manage to take a look at the failing tests?

**Comment by adrielparedes** [comment]
jenkins retest this

**Comment by kiereleaseuser2** [comment]
Build finished. 2759 tests run, 5 skipped, 2 failed.

**Comment by tiagobento** [comment]
@adrielparedes Yep, same here. No errors. We should be able to see Jenkins' build log. 

@mbiarnes Any idea why we're not getting a link on the PR to open it directly for the Errai repo?

**Comment by adrielparedes** [comment]
@tiagobento thanks a lot for running tests. @mbiarnes tell me how can I help you with this

**Comment by jhrcek** [comment]
Guys just a heads up (not gonna block merging this PR because of this).
When building this locally I noticed this:
```
[INFO] --- maven-dependency-plugin:2.8:unpack (unpack) @ errai-jsinterop-demo-app ---
[INFO] Configured Artifact: org.wildfly:wildfly-dist:10.0.0.Final:zip
[INFO] Unpacking /home/jhrcek/.m2/repository/org/wildfly/wildfly-dist/10.0.0.Final/wildfly-dist-10.0.0.Final.zip to /home/jhrcek/Devel/github.com/jhrcek/errai/errai-demos/errai-jsinterop-demo/errai-jsinterop-demo-app/target with includes "" and excludes ""
```
So just to keep consistency in eap versions being used some additional changes might be required.

**Comment by jhrcek** [comment]
Is this really finished?
screenshot from 2018-11-01 07-44-12

**Comment by adrielparedes** [comment]
@jhrcek not there anymore

**Comment by kiereleaseuser2** [comment]
Build finished. 2761 tests run, 5 skipped, 2 failed.

**Pull request merged**